src/index.js

```diff
@@ -299,6 +299,10 @@ const MainAppWithApolloProvider = () => {
     AppContainer = createAppContainer(MainTabNavigator);
   }
 
+  if (initialGlobalSettings.imageAspectRatio) {
+    consts.IMAGE_ASPECT_RATIO = parsedImageAspectRatio(initialGlobalSettings.imageAspectRatio);
+  }
```

Comment:
I would have expected this to be somewhere near line 192, where we set the settings depending on the content of the json file. what do you think?

Reply:
it is also working there. but i put it down here, because we already do something conditionally based on keys from global settings here (preparing the right navigation). so i decided to set the image aspect ratio from the settings key here as well.
around 192 there are setups for the app in general. in that case processing the server data for global settings and saving it to the local storage. i found it too detailed and out of context, to have a setter for a detailed functionality there.
we should wait for setting up the settings entirely (line 255) before working with the values. thats why i landed in 302.

Reply:
ok :+1: